crypto/bn/bn_prime.c

```diff
@@ -250,6 +250,17 @@ int ossl_bn_check_prime(const BIGNUM *w, int checks, BN_CTX *ctx,
     return bn_is_prime_int(w, checks, ctx, do_trial_division, cb);
 }
 
+/*
+ * Use this only for key generation. It always uses trial division. The number
```

Comment:
Use this only for key generation in a context where an attacker cannot observe the process.